assets/css/sass/utils/_mixins.scss

```diff
@@ -32,6 +32,15 @@
 	}
 }
 
+// Remove underline if element has a button class.
+@mixin notUnderlinedWhenButton() {
+	&.button,
+	&.components-button:not( .is-link ),
```

Comment:
`stylelint` complained about this so I fixed it ☑️ b1c3ef31ef6a5f6abfb00912870016a9ad83cdff

```
assets/css/sass/utils/_mixins.scss
 38:26  ✖  Expected single space after "("    selector-pseudo-class-parentheses-space-inside
 38:33  ✖  Expected single space before ")"   selector-pseudo-class-parentheses-space-inside
```

Reply:
Thank you! :tada: